src/plugins/saved_search/server/saved_objects/search.ts

```diff
@@ -33,44 +35,74 @@ export function getSavedSearchObjectType(
       },
     },
     mappings: {
+      dynamic: false,
       properties: {
-        columns: { type: 'keyword', index: false, doc_values: false },
         description: { type: 'text' },
```

Comment:
I'm not aware of anywhere we search on the `description` field, but it's indexed and seems like the kind of field that might be used for searching. Does anyone know if it's used in any requests anywhere, or should I go ahead and try to drop this one too?

Reply:
The SavedObjectsFinder component searches on title, but for some saved object types also on description IIRC. @Dosant you recently investigated this where does the code live that determines what fields get searched over from SavedObjectsFinder?

Reply:
`SavedObjectsFinder` by always searches by title and by description. Then types can add `additionalFields` to search on. 

https://github.com/elastic/kibana/blob/a1fccfd880041219f4195c5a1955f848d4efc0eb/src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx#L149

Reply:
@Dosant Thanks for clarifying! Not sure how I missed that one, but that answers my question 👍